README.md

```diff
@@ -0,0 +1,83 @@
+# Team awesome
+
+Team awesome is a time entry application. An user can add time entries and team members to the application. The time entries are stored in the timesheets on the first page of the application. The time entries are displayed by date and gives the user the information about the client, work hours and duration per day.
+
+The Team Members page displays all the team members of the company, their role, client and starting date.
+
+## Technologies
+
+---
+
+- Node.js
+- Next.js
+- React and JSX
+- JSON
+- Vercel
+- Storybook
+
+## Installation
+
+---
+
+Install the lastest version of Homebrew
+
+```bash
+/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"
+```
+
+Install Node Package Manager (NPM)
+
+```bash
+brew install node
+```
```

Comment:
You can also write multiple commands in one "bash field" with #instructions in between, like this:

```bash
#Install the lastest version of Homebrew
/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

#Install Node Package Manager (NPM)
brew install node

#etc 
brew something
```